warcit/warcit.py

```diff
@@ -240,7 +241,7 @@ def __init__(self, url_prefix, inputs,
             self.use_mapfile = True
             self.mapfile = mapfile
 
-        self.logfile = logfile
+        self.logfile = str(logfile + '.csv')
         self.use_logfile = False
         if self.logfile:
             self.use_logfile = True
```

Comment:
```suggestion
            self.use_logfile = True
            self.logfile = f'{self.logfile}.csv'
```

Could also be `self.logfile = '{}.csv'.format(self.logfile)` if we're trying to support Python < 3.6, but I don't think we have to worry about that at this point.